color_refinement.py

```diff
@@ -207,7 +207,9 @@ def is_isomorphisms(g: Graph, h: Graph) -> bool:
     """
     if preprocessing.is_tree(g):
         if preprocessing.is_tree(h):
-            return tree_isomorphism(g, h)
+            md = graph_to_modules(g + h)
+            graph_md, modular_iso = modules_to_graph_with_module_isomorphism(md) #TODO: graph_md?
```

Comment:
`graph_md` is de graaf gereconstrueerd uit `md`, dus de MD van graaf `g + h`.

Misschien is het equivalent en sneller om het volgende te doen:

```python
md_g = graph_to_modules(g)
md_h = graph_to_modules(h)
graph_md, modular_iso = modules_to_graph_with_module_isomorphism(md_g + md_h)
```

En nog sneller is om de `graph_md` helemaal niet te berekenen als je 'm niet nodig hebt (dus een methode maken die geen graaf instantieert). Maar heb je de MD niet nodig dan? In principe zijn in `g` en `h` geen MD, maar gewoon de uitgelezen graaf. Als uit MD blijkt dat er modules in de graaf zitten, dan wil de graaf decomponeren en dat resulteert misschien in een boom die je met tree refinement kunt oplossen.

Hier staat nu: gegeven een boom, zoek modules in de boom (zijn er meestal geen) en gebruik ze. Maar dat is dus niet sneller, want het was al een boom en die kunnen we al snel oplossen.